### ISSUE TITLE
tsconfig.json does not include project files causing issues with tsserver

### ISSUE THREAD

**Issue by Lewenhaupt** [ServerlessLife/serverless-spy#38]
@ServerlessLife As I've been starting my nvim journey I've encountered an issue with the tsconfig.json file. It only includes things under src. Unfortunately tsserver doesn't support setting an explicit tsconfig file and the issue has been open for over 5 years in the typescript repo...
I noticed that the projen file is a .bak file so I suspect it's no longer used? 
I think it will require some type-fixes in the project but it shouldn't take too long (found about 34 issues when I did a test).

Would you be ok with this? I'll try to make sure the projen file reflects the changes as well to make sure we don't break things if we have to rerun projen (but I suspect this would break a ton of things anyway).

**Comment by ServerlessLife** [comment]
I would be really grateful if you fixed those. The same goes for any bug you find.

Regarding `project.bak`, there is no need to keep it up to date. It is not up to date currently. When I configured the project with Projen, I had a lot of issues since the project is quite different from their initial template. That's why I decided to remove it (uses the `escape` command), but I left those files in case I ever change my mind. I suppose with using Projen escape hatches it would still be possible to use Projen, but it would require quite some work.

**Comment by Lewenhaupt** [comment]
> I would be really grateful if you fixed those. The same goes for any bug you find.
> 
> Regarding `project.bak`, there is no need to keep it up to date. It is not up to date currently. When I configured the project with Projen, I had a lot of issues since the project is quite different from their initial template. That's why I decided to remove it (uses the `escape` command), but I left those files in case I ever change my mind. I suppose with using Projen escape hatches it would still be possible to use Projen, but it would require quite some work.

Yeah I assumed that was the reason. Alright, then I'll just fix the tsconfig.json and the typing that are needed to have it pass 😊👍
